content/en/modules/git_github/index.md

```diff
@@ -35,19 +35,29 @@ The prerequisites to take this module are:
 Contact Pierre Bellec if you have questions on this module, or if you want to check that you completed successfully all the exercises.
 
 ## Resources
-This module was presented by [Elizabeth Dupre](https://elizabeth-dupre.com/#/) during the QLSC 612 course in 2020, with content adapted from the software carpentries by [Elizabeth Dupre](https://elizabeth-dupre.com/#/).
 
-All the tutorial notes are available [here](https://emdupre.github.io/git-course/).
+This module was written by [Elizabeth Dupre](https://elizabeth-dupre.com/#/) and Jenny Tou.
+
+Jenny Tou wrote the git concept module. She initially presented her workshops at Brigham and Women's Hospital in 2019. The video found here is a shortern version of the original presentation. Please contact her at sl.tou@mail.utoronto.ca for the complete presentation materials.
+
+[Elizabeth Dupre](https://elizabeth-dupre.com/#/) presented her material during the QLSC 612 course in 2020, with content adapted from the software carpentries by [Elizabeth Dupre](https://elizabeth-dupre.com/#/).
+
+All her tutorial notes are available [here](https://emdupre.github.io/git-course/).
 
 The video of her presentation is available below:
 <iframe width="560" height="315" src="https://www.youtube.com/embed/fBgxmpk9C4I" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 ## ATTENTION! 
 For the last lecture of the Introduction to Git and GitHub module ([Pull Requests](https://emdupre.github.io/git-course/06-pull-requests/)) please use [this repo](https://github.com/brainhack-school2022/more-papers) under Brainhack School 2022 Github organization to complete the tutorial instead of the one indicated in the tutorial (due to it is no longer in use.). Thank you! 
 
+Github authentication methods have changed! Please find the [latest documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/about-authentication-to-github).
+
+## Git Concept
+<iframe width="560" height="315" src="https://vimeo.com/822025017?share=copy" title="Vimeo video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
This is a great short presentation! Great addition to the full length course we already have.

Do you want to add a short description here?
Perhaps something like:
> This is a 10 minutes walk through of basic git concepts for people who wants a refresher!